pkg/kor/clusterrolebindings.go

```diff
@@ -0,0 +1,151 @@
+package kor
+
+import (
+	"bytes"
+	"context"
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	v1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/yonahd/kor/pkg/common"
+	"github.com/yonahd/kor/pkg/filters"
+)
+
+//go:embed exceptions/clusterrolebindings/clusterrolebindings.json
+var clusterRoleBindingsConfig []byte
+
+// Check if any valid service accounts exist in the ClusterRoleBinding
+func isUsingValidServiceAccountClusterScoped(serviceAccounts []v1.Subject, allServiceAccountNames map[string]map[string]bool) bool {
+	for _, sa := range serviceAccounts {
+		// Check if the service account exists in the specified namespace
+		if namespaceMap, namespaceExists := allServiceAccountNames[sa.Namespace]; namespaceExists {
+			if namespaceMap[sa.Name] {
+				return true
+			}
+		}
+	}
+	return false
+}
+
+func validateClusterRoleReference(crb v1.ClusterRoleBinding, clusterRoleNames map[string]bool) *ResourceInfo {
+	if crb.RoleRef.Kind == "ClusterRole" && !clusterRoleNames[crb.RoleRef.Name] {
+		return &ResourceInfo{Name: crb.Name, Reason: "ClusterRoleBinding references a non-existing ClusterRole"}
+	}
+
+	return nil
+}
+
+func processClusterRoleBindings(clientset kubernetes.Interface, filterOpts *filters.Options, opts common.Opts) ([]ResourceInfo, error) {
+	clusterRoleBindingsList, err := clientset.RbacV1().ClusterRoleBindings().List(context.TODO(), metav1.ListOptions{LabelSelector: filterOpts.IncludeLabels})
+	if err != nil {
+		return nil, err
+	}
+
+	clusterRoleNames, err := convertNamesToPresenseMap(retrieveClusterRoleNames(clientset, filterOpts))
+	if err != nil {
+		return nil, err
+	}
+
+	// Get all service accounts from all namespaces for cluster-scoped checking
+	allServiceAccountNames := make(map[string]map[string]bool)
```

Comment:
Maybe take the service account and it's namespace from the binding and search if it exists